bokehjs/src/less/tooltips.less

```diff
@@ -1,11 +1,13 @@
 @import "./_mixins.less";
 
 .bk-root {
+  --tooltipArrowColor: #909599; /* Gray of icons */
+
   @tooltipBorder: #e5e5e5;  /* Same border color used everywhere */
   @tooltipColor: white;
   @tooltipText: #2f2f2f;
 
-  @tooltipArrowColor: #909599;  /* Gray of icons */
+  @tooltipArrowColor: var(--tooltipArrowColor);
```

Comment:
This is not supported in IE11. You will have to either introduce a polyfill (if exists; to legacy bundles) or figure out a different way of styling this.

Reply:
Good catch. 

It looks like the [ie11-custom-properties package](https://www.npmjs.com/package/ie11-custom-properties) (MIT licensed) could do the trick, but I'm not sure how to add it to this project -- could you provide some guidance here? The example usage adds it as a `<script>` tag to HTML fwiw.

There are some alternative polyfill solutions in these SO threads but again, I'm not sure how to add them to the project 😕
- https://stackoverflow.com/q/46429937
- https://stackoverflow.com/q/45955538

An alternative solution I looked into was [`less.modifyVars()`](http://lesscss.org/usage/#using-less-in-the-browser-modify-variables) but I don't think it works for us since we compile to CSS before serving.